tests/test_pipe.py

```diff
@@ -98,18 +100,27 @@ def test_filter_target_tiles():
                 assert isinstance(tile, Tile)
         assert i == 4
 
+    pipe = tiles | PIPE.filter_target_tiles(overwrite=False)
+    i = 0
+    for tile in pipe.results():
+        if tile.status == "existing":
+            i += 1
+            assert isinstance(tile, Tile)
+    assert i == 4
+
 
 def test_upload_file():
     tiles = _get_subset_tiles()
-    with mock.patch.object(Tile, "upload", return_value=None):
-
+    # FIXME: I can't get this mock to work, despite trying to patch in many places
```

Comment:
is comment still valid?

Reply:
Afraid so. It's a pre-existing test I tried modifying to both improve and extend to cover existing tiles. But the mock as written doesn't work, and I can't figure out why.